theme/less/_header.less

```diff
@@ -0,0 +1,69 @@
+.header-top
+{
+  padding: 25px;
+}
+
+header label select
+{
+  width: 163px;
+  color: #3A7D11;
+  font-size: 24px;
+  border: 0;
+  opacity: 0.7;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background: url(/images/language-arrow.png) no-repeat right;
+}
+
+header a
+{
+  text-decoration: none;
+}
+
+.header-top a
+{
+  max-width: 190px;
+}
+
+header p
+{
+  display: inline;
+  opacity: 0.7;
+  color: #000000;
+  padding: 0 10px;
+}
+
+header img
+{
+ vertical-align: middle;
+ width: 32px;
+ height: auto;
+}
+
+#hamburger
```

Comment:
Mobile styles for current selector are located in `_navigation.less`, I think you intended either move styles from there here or vice-versa?

Reply:
Lets discuss this also on the context of merging `_navigation.less` with `_header.less`.